deploy_exchange/action.yml

```diff
@@ -0,0 +1,33 @@
+name: Deploy to Anypoint Exchange
+description: Deploy to Anypoint Exchange with Maven
+inputs:
+  connected_app_client_id:
+    description: CloudHub connected app client ID
+    required: true
+  connected_app_client_secret:
+    description: CloudHub connected app client secret
+    required: true
+  maven_settings_path:
+    description: Maven settings file path
+    required: true
+  mule_file_path:
+    description: Mule file path
+    required: true
+  cloudhub_business_group_id:
+    description: CloudHub business group ID
+  mule_runtime_version:
+    description: Mule runtime version
+    default: 4.4.0
+
+runs:
+  using: composite
+  steps:
+    - name: Deploy to Anypoint Exchange
+      shell: bash
+      run: |
+        mvn deploy --settings ${{ inputs.maven_settings_path }} -DskipTests -DbusinessGroupId=${CLOUDHUB_BUSINESS_GROUP_ID}
+      env:
+        CONNECTED_APP_CLIENT_ID: ${{ inputs.connected_app_client_id }}
+        CONNECTED_APP_CLIENT_SECRET: ${{ inputs.connected_app_client_secret }}
+        MULE_RUNTIME_VERSION: ${{ inputs.mule_runtime_version }}
+        CLOUDHUB_BUSINESS_GROUP_ID: ${{ inputs.cloudhub_business_group_id }}
```

Comment:
Would it make more sense to keep `env` above `run`?

Reply:
From my understanding, this is the ENV to run that command, where the `-DbusinessGroupId` is the param that gonna send to the pom.xml

Reply:
I'm checking this topic FYI

Reply:
Seem we can remove that @toby-thanathip - I removed it in https://github.com/nimblehq/mulesoft-actions/pull/12/commits/081ef4b44d2a33f3f9357dcc299d67f52813b7e1 and now will checking on the app side

Reply:
@toby-thanathip I tested on this https://github.com/Jollibee-Foods-Corporation/jfc-global-api-common-module/actions/runs/7165702346/job/19508191080

And we can deploy the new module to Exchange

![image](https://github.com/nimblehq/mulesoft-actions/assets/11751745/7d7d6269-37c6-4fe8-a03a-8cbea9cb10c6)